locksmith/openapi.yml

```diff
@@ -48,37 +48,42 @@ paths:
           description: OK
   /v2/api/{network}/locks/{lockAddress}/keys:
     get:
-    parameters:
-      - in: path
-        name: network
-        required: true
-        schema:
-          type: integer
-        description: Network id
-      - in: path
-        name: lockAddress
-        required: true
-        schema:
-          type: string
-        description: Lock address
       description: List of lock keys by network with additional metadata if the caller is the lockManager
-      requestBody:
-        content:
-          application/json:
-            schema:
-              properties:
-                query:
-                  type: string
-                filterKey:
-                  type: string
-                expiration:
-                  type: string
-                page: 
-                  type: integer
```

Comment:
@kalidiagne I think you added this earlier. When declaring query params, it should in the parameters - not the request body. Request body is for actual body you send. On get request, you cannot so it results in validation error.